Integrations/BitDam/Pipfile

```diff
@@ -0,0 +1,23 @@
+[[source]]
+name = "pypi"
+url = "https://pypi.org/simple"
+verify_ssl = true
+
+[dev-packages]
+pylint = "*"
+pytest = "*"
+pytest-mock = "*"
+requests-mock = "*"
+flake8 = "*"
+
+[packages]
```

Comment:
remove the not needed ones from here

Reply:
Not sure what are the not needed ones, as this file was automatically created. Let's discuss